examples/browser/test.js

```diff
@@ -8,6 +8,14 @@
   var g = { year: year }
   var n = {}
 
+  function duration (ms) {
+    var days = (ms / 86400000) | 0
+    var hours = (ms - (days * 86400000)) / 3600000 | 0
```

Comment:
Minor: it is probably worth defining "constants" (as much as there can be constants in ES 5) for these numbers.